src/UpsilonGC/UpsilonGCHeap.cpp

```diff
@@ -414,7 +417,7 @@ Object * UpsilonGCHeap::Alloc(gc_alloc_context * acontext, size_t size, uint32_t
 		gcToCLR->RestartEE(true);
 	}
 	int beginGap = 24;
-	uint8_t* newPages = (uint8_t*)VirtualAlloc(NULL, GrowthSize, MEM_RESERVE | MEM_COMMIT, PAGE_READWRITE);
+	uint8_t* newPages = (uint8_t*)VirtualAlloc(nullptr, GrowthSize, MEM_RESERVE | MEM_COMMIT, PAGE_READWRITE);
```

Comment:
Does it compile? I wonder from where `VirtualAlloc` come from in case of Linux?

Reply:
VirtualAlloc is available at coreclr/src/pal/inc/pal.h

So cmake option is:

```
cmake .. -DCMAKE_C_COMPILER=clang \
    -DCMAKE_CXX_COMPILER=clang++ \
    -DCORECLR_FOR_CUSTOMGC="/home/loongson/zhaixiang/coreclr-mips64-dev"
```